src/styles/base.scss

```diff
@@ -21,17 +22,19 @@ a {
 
 /** Header Styles **/
 h1, h2, h3, h4, h5, h6 {
-  &.title {
-    &.heading-with-background {
-      margin-bottom: 1rem;
-      padding-left: 1rem;
-      padding-right: 1rem;
-      background-color: $ghost-gray;
-    }
-    &.heading-with-contrast {
-      margin-bottom: .5rem;
-      border-bottom: 10px solid $ben-franklin-blue;
-    }
+  font-family: $family-secondary;
+  font-weight: $weight-normal;
+  &.heading-with-background {
```

Comment:
We've also un-nested these two classes to that they are not depended on the .title class